### PR TITLE
#7007: Upgrade CesiumJs to latest stable version

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

This PR upgrades Cesium to the version 1.90 and it includes following changes:

- remove external script and css related to cesium and cesium-navigation now included as package
- update map preview component of print plugin to import map lib dynamically
- refactor of build process related to cesium

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Other... Please describe: library update

## Issue

**What is the current behavior?**

#7007

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] Yes, and I documented them in migration notes

### CODE REVIEW

**Review by offtherailz — COMMENTED** [review]
It looks a big simplification and I like it a lot.
I'd like to know if it is possible to avoid to add to webpack custom build entries for Cesium, and use instead the dynamic import to allow to make webpack build it as a separate module without explit work for us. 

A support is already present in Map.jsx. https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/Map.jsx#L439 

pluginsCreator should do a dynamic import of the library before starting to use it.  

This to simplify even more, if possible. 

Please let me know.

File: docs/developer-guide/mapstore-migration-guide.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Downstream project should update following configurations:
 -<link rel="stylesheet" href="libs/cesium-navigation/cesium-navigation.css" />
 ```
 
-- update the `cesiumBaseUrl` parameter in the webpack dev/prod config to point to the correct location of the compiled cesium folder. All the cesium assets are located in the dist target folder under (default `dist/cesium`). This is needed only for project with specific `publicPath` that are not using the default one.
+- This step is needed only for custom project with a specific `publicPath` different from the default one (default `dist/cesium`). Update the `cesiumBaseUrl` parameter in the webpack dev/prod config to point to the correct location of the cesium static assets, widgets and workers folder.
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:151]:
```suggestion
- This step is needed only for custom project with a specific `publicPath` different from the default one. In this case you may need to specify what folder deliver the  cesium build ( by default `dist/cesium`). To do that, you can add the  `cesiumBaseUrl` parameter in the webpack dev and prod configs to the correct location of the cesium static assets, widgets and workers folder.
```

**Review by offtherailz — APPROVED** [review]

**Comment by allyoucanmap** [comment]
Hi @ElenaGallo kindly test this issue in DEV, thanks

**Comment by ElenaGallo** [comment]
@allyoucanmap  Testing this PR I found these two problems:

1_ Adding a new background to the demo map, with 3D mode on, couse an error and the gray error page appears. 

> GET https://dev-mapstore.geosolutionsgroup.com/mapstore/proxy/?url=https%3A%2F%2F1maps.geo-solutions.it%2Fgeoserver%2Fgwc%2Fservice%2Fwmts%3Fservice%3DWMTS%26REQUEST%3DDescribeDomains%26version%3D1.0.0%26layer%3Ds2cloudless%253As2cloudless%26tileMatrixSet%3DEPSG%253A4326%26expandLimit%3D10 400 (400)




2_ Opening a new map and adding a new background, with 3D mode on, cause an error on console:

> Access to XMLHttpRequest at 'https://gs-stable.geo-solutions.it/geoserver/gwc/service/wmts?service=WMTS&REQUEST=DescribeDomains&version=1.0.0&layer=ny_layergroup&tileMatrixSet=EPSG:4326&expandLimit=10&authkey=93682457-9a58-420f-9568-57c7380bd18b' from origin 'https://dev-mapstore.geosolutionsgroup.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
> GET https://gs-stable.geo-solutions.it/geoserver/gwc/service/wmts?service=WMTS&REQUEST=DescribeDomains&version=1.0.0&layer=ny_layergroup&tileMatrixSet=EPSG:4326&expandLimit=10&authkey=93682457-9a58-420f-9568-57c7380bd18b net::ERR_FAILED 500

**Comment by allyoucanmap** [comment]
> 2_ Opening a new map and adding a new background, with 3D mode on, cause an error on console:
> 
> Access to XMLHttpRequest at 'https://gs-stable.geo-solutions.it/geoserver/gwc/service/wmts?service=WMTS&REQUEST=DescribeDomains&version=1.0.0&layer=ny_layergroup&tileMatrixSet=EPSG:4326&expandLimit=10&authkey=93682457-9a58-420f-9568-57c7380bd18b' from origin 'https://dev-mapstore.geosolutionsgroup.com/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
> GET https://gs-stable.geo-solutions.it/geoserver/gwc/service/wmts?service=WMTS&REQUEST=DescribeDomains&version=1.0.0&layer=ny_layergroup&tileMatrixSet=EPSG:4326&expandLimit=10&authkey=93682457-9a58-420f-9568-57c7380bd18b net::ERR_FAILED 500

@ElenaGallo @tdipisa this one is not an issue related the 3D because it's happening also on OpenLayers. The first request is a failed request because of CORS and the second one is the timeline request and it is failing probably because that layer does not provide time dimension (could be a layer group?)

**Pull request merged**